build.xml

```
@@ -42,7 +43,7 @@
 
   <target name="compile">
     <mkdir dir="${build}"/>
-    <javac classpathref="build.class.path" debug="on" srcdir="${src}" destdir="${build}" source="1.5" target="1.5"/>
+    <javac classpathref="build.class.path" debug="on" srcdir="${src}" destdir="${build}" source="1.6" target="1.6"/>
```

```suggestion
    <javac classpathref="build.class.path" debug="on" srcdir="${src}" destdir="${build}" source="1.5" target="1.5"/>
```
Let's try 1.5 for now.